### ISSUE TITLE
Metric scale for monocular image

### ISSUE THREAD

**Issue by chapa17** [TRI-ML/packnet-sfm#206]
Hello,

I am trying to run the code on my own dataset. I am using a monocular camera. The depth map when reconstructed to 3D point cloud has ground at some elevation which is different for every point. I am aware that I need to scale the point cloud. But I am not sure how to obtain the metric scale for depth.

It would be really great if someone can provide some hint to this.

Thank you!

**Comment by VitorGuizilini-TRI** [comment]
The monocular pointclouds should be accurate up to a single scale factor, so it is strange that each ground point has a different elevation.  They should be consistent, just not metrically accurate. To obtain metric scale you can either use GT depth for supervision or our velocity weak supervision in addition to self-supervision.  I hope that helps!

**Comment by chapa17** [comment]
Yes, I am trying to use the pre-trained model with velocity supervision i.e. [PackNet, Self-Supervised Scale-Aware, 192x640, CS → K](https://tri-ml-public.s3.amazonaws.com/github/packnet-sfm/models/PackNet01_MR_velsup_CStoK.ckpt). I hope it is the same you are trying to point at. 

Also I get the depth values as high as 85. Can you please confirm the unit of this values (mm or cm)?
